js/src/Debug/Widget/HTTPRequests.vue

```diff
@@ -0,0 +1,298 @@
+<script setup>
+import {computed, ref, watch} from "vue";
+
+    const props = defineProps({
+        initial_request: {
+            type: Object,
+            required: true
+        },
+        ajax_requests: {
```

Comment:
Props are passed by ref so when a new AJAX request is added to the array by the main toolbar component, this child component will be aware of the change and it will be able to trigger adding the new request to the table.